hack/test-integration.sh

```diff
@@ -2,151 +2,8 @@
 STARTTIME=$(date +%s)
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-os::build::setup_env
+# build the test executable and make sure it's on the path
+"${OS_ROOT}/hack/build-go.sh" "test/integration/integration.test"
+os::util::environment::update_path_var
 
-export API_SCHEME="http"
-export API_BIND_HOST="127.0.0.1"
-os::cleanup::tmpdir
-os::util::environment::setup_all_server_vars
-
-function cleanup() {
-	return_code=$?
-
-	# this is a domain socket. CI falls over it.
-	rm -f "${BASETMPDIR}/dockershim.sock"
-
-	os::test::junit::generate_report
-	os::cleanup::all
-	os::util::describe_return_code "${return_code}"
-
-	exit "${return_code}"
-}
-trap "cleanup" EXIT
-
-export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
-
-# Internalize environment variables we consume and default if they're not set
-package="${OS_TEST_PACKAGE:-test/integration}"
-name="$(basename ${package})"
-dlv_debug="${DLV_DEBUG:-}"
-verbose="${VERBOSE:-}"
-junit_report="${JUNIT_REPORT:-}"
-
-if [[ -n "${JUNIT_REPORT:-}" ]]; then
-	export JUNIT_REPORT_OUTPUT="${LOG_DIR}/raw_test_output.log"
-	rm -rf "${JUNIT_REPORT_OUTPUT}"
-fi
-
-# CGO must be disabled in order to debug
-if [[ -n "${dlv_debug}" ]]; then
-	export OS_TEST_CGO_ENABLED=0
-fi
-
-# build the test executable
-if [[ -n "${OPENSHIFT_SKIP_BUILD:-}" ]]; then
-  os::log::warning "Skipping build due to OPENSHIFT_SKIP_BUILD"
-else
-	"${OS_ROOT}/hack/build-go.sh" "${package}/${name}.test"
```

Comment:
Ah, this is because we run integration.test outside of hack/env.  Hrm, will fix that later.